src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -43,16 +43,18 @@ public bool TryGoToDefinition(Document document, int position, CancellationToken
         {
             // First try to compute the referenced symbol and attempt to go to definition for the symbol.
             var symbolService = document.GetLanguageService<IGoToDefinitionSymbolService>();
-            var (symbol, span) = symbolService.GetSymbolAndBoundSpanAsync(document, position, cancellationToken).WaitAndGetResult(cancellationToken);
+            var (symbol, _) = symbolService.GetSymbolAndBoundSpanAsync(document, position, cancellationToken).WaitAndGetResult(cancellationToken);
             if (symbol == null)
             {
                 return false;
             }
 
             var isThirdPartyNavigationAllowed = IsThirdPartyNavigationAllowed(symbol, position, document, cancellationToken);
 
-            return GoToDefinitionHelpers.TryGoToDefinition(symbol,
+            return GoToDefinitionHelpers.TryGoToDefinitionOrParentDefinition(symbol,
```

Comment:
what does the term 'parent' mean in this context?  It sounds like it would be better phrases as TryGoToDefinitionOrRelatedDefinitions.